icgcget/commands/download.py

```diff
@@ -186,7 +185,9 @@ def get_manifest(self, manifest, file_ids, api_url, repos, portal):
 
     def move_files(self, staging, output):
         for staged_file in os.listdir(staging):
-            if staged_file != "state.json":
+            if staged_file == "cidfile":
+                os.remove(os.path.join(staging, staged_file))
+            elif staged_file != "state.json":
                 try:
```

Comment:
If I'm understanding this correctly, your outer `try`/`except` block is actually handling errors from the inner `except`.  This forces your eyes to bounce to the end and back up to the top to understand the flow.  Do this instead:

```python
try:
    ...
except shutil.Error:
    try:
        ....
     except OSError:
        ....
```

Also you need to check the OSError error number. If I were to remove the `.staging` file before this ran, it would still throw an OSError complaining about no such file or directory, but your message would indicated it is a permission issue. 

https://docs.python.org/2/library/errno.html